lib/mongodb.js

```diff
@@ -1013,6 +1013,31 @@ MongoDB.prototype.fromDatabaseToPropertyNames = function(model, data) {
   return this.convertColumnNames(model, data, 'property');
 };
 
+MongoDB.prototype.fromFieldFilterToDatabaseNames = function(model, data) {
```

Comment:
In retrospect, what you said made me realize this is probably never called directly, so it doesn't need to
be described in the README. However, it should still be documented for future maintenance, and to provide autocompletion assistance within JSDoc-compatible IDEs/plugins:

```js
/**
 * Describe what the function does here
 * @param {Model} model The LoopBack model
 * @param {PutTheTypeOfDataHere} data Put a description here for what the data is.
 * @returns {PutTheTypeOfDataHere} The database name data.
 */
MongoDB.prototype.fromFieldFilterToDatabaseNames = function(model, data) {
// etc...
```

Reply:
I'll do that and the unit tests tomorrow when I'm back behind my main development machine.